app/views/doc/ext.html.erb

```diff
@@ -54,7 +48,7 @@
           </p>
         </li>
         <li>
-          <h4><a href="http://hoth.entp.com/output/git_for_designers.html">Git for Designers</a></h4>
+          <h4><a href="https://web.archive.org/web/20150301060509/http://hoth.entp.com/output/git_for_designers.html">Git for Designers</a></h4>
```

Comment:
This matches our "fix" for another dead link earlier in the list. We may want to revisit the idea of linking to the Internet Archive, though, as it means the pages are not really maintained. In theory advice about using Git should hold true forever, but things _do_ change in the Git world over time (better ways of doing things, best practices, etc). We may be better off directing users to more up-to-date content.

That said, I think this whole list could probably stand to be audited (and if there are gaps between the manpages and the book content, we should consider filling them on this site). Let's go with this as a link-fix for now, and deal with revising the whole list later.